rest/encoding.py

```diff
@@ -50,7 +54,11 @@ def encode(self, dct):
       root = self.encode_list(dct)
     else:
       root = self.encode_dict(dct)
-    return ElementTree.tostring(root, encoding='UTF-8')
+    if sys.version_info[0] >= 3:
+      return b"<?xml version='1.0' encoding='UTF-8'?>\n" + \
```

Comment:
Was this an automated change or did you make this manually?

Reply:
manual - this does not return the <?xml version='1.0' encoding='UTF-8'?> in python 3

Reply:
Im a little confused about this, why would we not need an opening xml tag in py2? Are you saying the old version added that automatically, can we just be explicit regardless of version?

Reply:
i'm saying the `tostring` call in py2 will add this automatically